packages/lesswrong/components/common/ErrorBoundary.jsx

```diff
@@ -0,0 +1,42 @@
+import { Components, registerComponent } from 'meteor/vulcan:core';
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+
+const styles = theme => ({
+  errorText: {
+    color: theme.palette.error.main,
+  }
+})
+
+
+class ErrorBoundary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { error: false };
+  }
+
+  componentDidCatch(error, info) {
+    this.setState({ error: error.toString() });
+  }
+
+  render() {
+    const { classes } = this.props;
+    if (this.state.error) {
+      // You can render any custom fallback UI
+      return <Typography
```

Comment:
You can just throw a className into the Typography, instead of adding an additional span (at least I am pretty sure that's doable).